README.md

```diff
@@ -104,28 +66,17 @@ See the full list of [our backers](https://mui.com/material-ui/discover-more/bac
 
 For how-to questions that don't involve making changes to the code base, please use [Stack Overflow](https://stackoverflow.com/questions/) instead of GitHub issues.
 
-## Examples
-
-Our documentation features [a collection of example projects](https://github.com/mui/material-ui/tree/master/examples).
-
-## Premium templates
-
-You can find complete templates and themes in the [MUI Store](https://mui.com/store/?utm_source=docs&utm_medium=referral&utm_campaign=readme-store).
-
 ## Contributing
 
 Read the [contributing guide](/CONTRIBUTING.md) to learn about our development process, how to propose bug fixes and improvements, and how to build and test your changes.
 
-Contributing is about more than just issues and pull requests!
-There are many other ways to [support MUI](https://mui.com/material-ui/getting-started/faq/#mui-is-awesome-how-can-i-support-the-project) beyond contributing to the code base.
-
 ## Changelog
 
-The [changelog](https://github.com/mui/material-ui/releases) is regularly updated to reflect what's changed in each new release.
+The [changelog](https://github.com/mui/base-ui/releases) is regularly updated to reflect what's changed in each new release.
 
 ## Roadmap
 
-Future plans and high-priority features and enhancements can be found in our [roadmap](https://mui.com/material-ui/discover-more/roadmap/).
+Future plans and high-priority features and enhancements can be found in [our roadmap](https://mui.com/base-ui/getting-started/roadmap/).
```

Comment:
I'll add this page for the Base UI docs now that we made all GitHub projects public!

Reply:
Can we link the GH project here instead?

Reply:
I think that makes sense, however, I was planning to add a similar Roadmap page to the Base docs as we have on the Joy UI docs, which includes a little bit of rationale about the roadmap (and it then links to the GitHub project).